guides/common/modules/proc_configuring-an-external-dhcp-server.adoc

```diff
@@ -163,8 +163,10 @@ Clients are configured using NFSv3.
 +
 [options="nowrap"]
 ----
-# firewall-cmd --zone public --add-service mountd \
-&& firewall-cmd --zone public --add-service rpc-bind \
-&& firewall-cmd --zone public --add-service nfs \
-&& firewall-cmd --runtime-to-permanent
+# firewall-cmd \
+--add-service mountd \
+--add-service nfs \
+--add-service rpc-bind \
```

Comment:
Non-blocking, but since there are two steps with firewall-cmd immediately one after another, it is sufficient to only call `--runtime-to-permanent` once, see https://github.com/theforeman/foreman-documentation/pull/2743#discussion_r1479707148